recipes-browser/wpewebkit/wpewebkit_2.21.91.bb

```diff
@@ -0,0 +1,11 @@
+require wpewebkit.inc
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+SRC_URI = "https://wpewebkit.org/releases/${PN}-${PV}.tar.xz \
+           file://0001-REGRESSION-r217927-WPE-2.22-GSTREAMER_GL-Video-appea.patch"
+
+SRC_URI[md5sum] = "08778a95eba2de141d8aaff3d802c811"
+SRC_URI[sha256sum] = "e75b1ca225fa9303017e954b6203e4f1750a7566fa852443e6fd574fd5f926b8"
+
+DEPENDS += " libwpe"
+RCONFLICTS_${PN} = "wpebackend (< 1.0) wpebackend-fdo (< 1.0)"
```

Comment:
@philn I am getting:
`Problem: package wpewebkit-2.21.91-r0.armv7vet2hf_neon conflicts with wpebackend < 1.0 provided by libwpebackend-0.1-0-0.1-r0.armv7vehf_neon` Is there a recipe fot the `wpebackend` 1.0? I can only see `0.1` version.

Reply:
There is a recipes-browser/libwpe/libwpe_1.0.0.bb ... WPEBackend was renamed to libwpe just before the 1.0 release.

Reply:
Or maybe the idea is that the wpebackend should not be installed at all and the libwpe should be used instead

Reply:
Yeah, just figured that out. Thanks